controllers/secretagentconfiguration_controller.go

```diff
@@ -18,6 +18,7 @@ package controllers
 
```

Comment:
you may be better off applying your changes to the file manually rather than trying to merge